litestar/cli/commands/core.py

```diff
@@ -71,24 +71,25 @@ def info_command(app: Litestar) -> None:
 )
 @option("--host", help="Server under this host", default="127.0.0.1", show_default=True)
 @option(
+    "-fd",
     "--fd",
     "--file-descriptor",
     help="Bind to a socket from this file descriptor.",
     type=int,
     default=None,
     show_default=True,
 )
-@option("--uds", "--unix-domain-socket", help="Bind to a UNIX domain socket.", default=None, show_default=True)
-@option("--debug", help="Run app in debug mode", is_flag=True)
-@option("--pdb", "use_pdb", help="Drop into PDB on an exception", is_flag=True)
-@option("--reload-dir", help="Directories to watch for file changes", multiple=True)
+@option("-uds", "--uds", "--unix-domain-socket", help="Bind to a UNIX domain socket.", default=None, show_default=True)
+@option("-d", "--debug", help="Run app in debug mode", is_flag=True)
+@option("-pdb", "--use_pdb", help="Drop into PDB on an exception", is_flag=True)
```

Comment:
```suggestion
@option("-pdb", "--use-pdb", help="Drop into PDB on an exception", is_flag=True)
```
Maybe consider doing this since we use `-` for the others?

Reply:
The `use_pdb` was using an underscore because it wasn’t specifying a flag, but the argument name for click. As a flag, this should use a dash.

Reply:
I'm not sure I follow why there's a distinction between argument vs flag as user.  Wouldn't we want the CLI flags to be consistent or is there an expected naming convention that I've seemingly forgotten about (which is very possible)?